tests/modules/programs/rbw/settings.nix

```diff
@@ -0,0 +1,33 @@
+{ pkgs, ... }:
+let
+  expectedRegex = ''
+    {
+      "base_url": "bitwarden.example.com",
+      "email": "name@example.com",
+      "identity_url": "identity.example.com",
+      "lock_timeout": 300,
+      "pinentry": "/nix/store/.*-pinentry"
+    }
+  '';
+in {
+  config = {
+    programs.rbw = {
+      enable = true;
+      settings = {
+        email = "name@example.com";
+        base_url = "bitwarden.example.com";
+        identity_url = "identity.example.com";
+        lock_timeout = 300;
+        pinentryFlavor = "tty";
+      };
+    };
+
+    nixpkgs.overlays = [ (import ./overlay.nix) ];
+
+    nmt.script = ''
+      assertFileRegex \
+        home-files/.config/rbw/config.json \
+        '${expectedRegex}'
+    '';
+  };
+}
```

Comment:
Can you add an overlay for the selected pinentry method?

Overlay:
```
self: super: { pinentry.some-flavor = self.writeScriptBin "pinentry-some-flavor" '' // { outPath = "@pinentry@"; }; }
```

Then you can use `assertFileContent` instead of `assertFileRegex`:

```
expectedContent = ''
  stuff above
  "pinentry": "@pinentry@"
'';
```

Reply:
Okay I figured a way around it.

Also, the option must be named `pinentry` in the configuration file, not `pinentryFlavor` as I had pasted by error.